### PR TITLE
CR-1049331 HBM temp missing from xbutil query Temperature section

### PR DESCRIPTION
**Pull request by xdavidz**
requested by marking, just simply add temp[0] as hbm temp. 

Test Results:
```
xbutil query
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Temperature(C)                                                                  
PCB TOP FRONT   PCB TOP REAR    PCB BTM FRONT   VCCINT TEMP                     
36              31              N/A             41                              
FPGA TEMP       TCRIT Temp      FAN Presence    FAN Speed(RPM)                  
39              35              P               N/A                             
QSFP 0          QSFP 1          QSFP 2          QSFP 3                          
N/A             N/A             N/A             N/A                             
HBM TEMP                                                                        
34  

xbutil dump
        "physical":                                                      
        {                                                                
            "thermal":                                                   
            {                                                            
                "pcb":                                                   
                {                                                        
                    "top_front": "36",                                   
                    "top_rear": "31",                                    
                    "btm_front": "0"                                     
                },                                                       
                "fpga_temp": "39",                                       
                "tcrit_temp": "35",                                      
                "fan_presence": "P",                                     
                "fan_speed": "0",                                        
                "vccint_temp": "41",                                     
                "cage":                                                  
                {                                                        
                    "temp0": "0",                                        
                    "temp1": "0",                                        
                    "temp2": "0",                                        
                    "temp3": "0"                                         
                },                                                       
                "hbm_temp": "34"                                         
            },
```

### CODE REVIEW

**Comment by AShivangi** [comment]
What does this print when the HBM bank isn't available?

**Comment by xdavidz** [comment]
> 
> 
> What does this print when the HBM bank isn't available?

XCL_NO_SENSOR_DEV(~(0U)) , I think it is in sync with what HBM is unavailable.

**Comment by AShivangi** [comment]
Hmm. I'm wondering if we should expose a sysfs node. If I'm not mistaken the driver has this value in xcl_sensor struct as hbm_temp0 and we should then call "sysfs_get_sensor" like all other sensor value calls

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.h [hunk 1]
```diff
@@ -1051,6 +1053,8 @@ class device {
              << std::setw(16) << sensor_tree::get_pretty<unsigned int>( "board.physical.thermal.cage.temp1" )
              << std::setw(16) << sensor_tree::get_pretty<unsigned int>( "board.physical.thermal.cage.temp2" )
              << std::setw(16) << sensor_tree::get_pretty<unsigned int>( "board.physical.thermal.cage.temp3" ) << std::endl;
+        ostr << std::setw(16) << "HBM TEMP" << std::endl;
+        ostr << std::setw(16) << sensor_tree::get<std::string>( "board.physical.thermal.hbm_temp") << std::endl;
```
Comment (AShivangi) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.h]: Use get_pretty<unsigned int>instead

**Comment by xdavidz** [comment]
retest this please

**Comment by xdavidz** [comment]
retest this please

**Pull request merged**